BasicGUI/Interpolator.cpp

```diff
@@ -71,41 +44,38 @@ QColor Interpolator::interpolatorSetLedColor(QVector<QPointF> vector_points)
     return color = color.fromRgb(red_component.level, green_component.level,
                                  blue_component.level);
 }
-/**
- *      @brief Function Interpolator::interpolatorTransform
- *       creates a vector points, which are contained in actually checked
- *       rectangle
- */
-
-QVector<QPointF> Interpolator::interpolatorTransform(QPoint center_of_rot,
-                                                     QRect rect,
-                                                     float deg_angle)
+//////////////////////////////////////////////////////////////////
+QVector<QPointF> Interpolator::interpolatorTransform(QPoint a_center_of_rot,
+                                                     QRect a_rect,
+                                                     float a_deg_angle)
```

Comment:
Please consider extracting math to separate object like `Rotate` and provide it by base class like `Transform` where `operator()(QPointF)` will be responsible to compute transformed point.

Reply:
As an alternative you could pass `Rotate` by `std::function<QPointF(QPointF)>`